### PR TITLE
Porting from bootstrap.css to material.css

### PR DESCRIPTION
**Pull request by anshulkgupta**
Major changes are on the team.html page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,61 @@
...
-```
\ No newline at end of file
+#! /bin/bash
+virtualenv --no-site-packages .
+source bin/activate
+pip install -r requirements.txt
+```
+
+## Material Design vs. Bootstrap
+
```
Comment (jackiehluo) [inline README.md]: Is this explanation necessary? Maybe just the part about using Material Design?

File: data/team.json [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+				"name": "James Xue",
+				"pic": "static/img/people/james.jpg",
+				"title": "Editor-in-Chief of Impulse; SEAS '17, Computer Science.",
+				"bio": "James started with CORE Impulse since it was just \"the blog\" back in 2013, helping to  grow a talented team, a large audience, and a site with some awesome content. He's worked at a tech startup in Hong Kong, a research lab in Tokyo, startups in NYC, and an IT consultancy in London. He also likes Game of Thrones!",
+				"twitter": "http://twitter.com/therealjamesxue"
+			},
+			{
+				"name": "Jamie Song",
+				"pic": "static/img/people/jamie.jpg",
+				"title": "Secretary<br>SEAS '17, Computer Science.",
```
Comment (jackiehluo) [inline data/team.json]: <br> in text!

File: data/team.json [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+			{
+				"name": "Kat Moon",
+				"pic": "static/img/people/kat.jpg",
+				"title": "CC '17, Creative Writing.",
+				"twitter": "http://twitter.com/katxmoon"
+			},
+			{
+				"name": "Kimberly Hou",
+				"pic": "static/img/people/kimberly.jpg",
+				"title": "Vice President<br>CC '16, Neuroscience.",
```
Comment (jackiehluo) [inline data/team.json]: `<br>` in text!

File: data/team.json [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+				"twitter": "http://twitter.com/kimberlyjhou"
+			},
+			{
+				"name": "Max Rosenberg",
+				"pic": "static/img/people/max.jpg",
+				"title": "Treasurer<br>CC '18, Economics."
```
Comment (jackiehluo) [inline data/team.json]: `<br>` in text!

File: data/team.json (hunk 2 above)
Comment (anshulkgupta) [inline data/team.json]: need better way of formatting this but I don't have the right answer!

Comment (jackiehluo) [inline data/team.json]: Oh, yeah, I meant that it shows as `<br>` and not a line break on the website!

Comment (anshulkgupta) [inline data/team.json]: fixed it! standy for changes on my fork.

**Pull request merged**